.github/workflows/scripts/firebase.json

```
@@ -12,9 +13,15 @@
     "auth": {
       "port": "9099"
     },
+    "storage": {
+      "port": "9080"
```

Default port for storage emulator is 9199 https://github.com/firebase/firebase-js-sdk/blob/50860d794bfe711143b5d287e1042eeb5187e47b/packages/rules-unit-testing/src/api/index.ts#L48
```suggestion
      "port": "9199"
```